tests/bootstrap.php

```diff
@@ -55,5 +55,28 @@ static function () {
 	1
 );
 
+
+$testsuite_count = array_count_values( $_SERVER['argv'] )['--testsuite'];
```

Comment:
Rather than inventing our own CLI args, I think we could just use the `--group` or `--filter` options for PHPUnit to run specific suites and load all of the plugins like do for modules above.

Reply:
testsuite is a supported arg in PHPUnit